core/src/test/java/org/apache/calcite/sql/type/SqlTypeFactoryTest.java

```diff
@@ -112,6 +121,26 @@ private void checkPrecision(int p0, int p1, int expectedMax,
     assertThat(SqlTypeUtil.comparePrecision(p1, p1), is(0));
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-2464">[CALCITE-2464]
+   * Allow to set nullability for columns of structured types</a>. */
+  @Test
+  public void createTypeWithNullability() {
```

Comment:
Shouldn't the name of the test reflect the fact that it only tests stuctured types?